CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+# Contribution guide
+
+The codebase is maintained using the "contributor workflow" where anyone can
+contribute patch proposals using
+[pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+
+To contribute a patch, the workflow is as follows:
+
+1. Fork the repository.
+2. Create a topic branch.
+3. Commit patches to the branch.
+4. Push changes to the fork.
+5. Create a pull request to merge the branch of the fork into this repository.
+6. If you had someone specifically in mind, ask them to review the pull request.
+Otherwise, just wait for a code review: most members with merge permissions
+receive notifications for newly created pull requests.
+7. Address review comments, if any.
+8. Merge and submit the pull request. If you don't have merge permissions,
+a reviewer will do it for you.
+
+> **NOTE:** Before starting any coding work, please make sure to discuss the
+> issue first to agree on the optimal way of solving a particular problem.
+> If no issue exists, create one unless the change is trivial, such as
+> correcting typos and code code formatting.
+>
+> If you don't know where to start, look for issues labeled
+> with `good-first-issue`.
```

Comment:
could link https://github.com/digitalbitbox/bitbox-wallet-app/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue